spring-boot-starter/example-autodeployment/pom.xml

```
@@ -7,8 +7,8 @@
   <version>0.0.1-SNAPSHOT</version>
 
   <properties>
-    <spring.boot.version>2.2.5.RELEASE</spring.boot.version>
-    <camunda.spring.boot.starter.version>7.13.0</camunda.spring.boot.starter.version>
+    <spring.boot.version>2.3.1.RELEASE</spring.boot.version>
```

Spring Boot 2.3.4.RELEASE is out, let's use it if it works (should work fine but please test it shortly 😃)
```suggestion
    <spring.boot.version>2.3.4.RELEASE</spring.boot.version>
```